PetAdoptions/keycloak-setup.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+
+echo "This script sets up keycloak related resources for Amazon Managed Grafana SAML authentication."
+
+export CLUSTER_NAME=PetSite
+export WORKSPACE_NAME=demo-amg
+export KEYCLOAK_NAMESPACE=keycloak
+export KEYCLOAK_REALM_AMG=amg
```

Comment:
We probably need to take these as inputs and fail the script if they are not provided or use overridable default values. People can decide to name their MG workspace differently for example.